android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -97,6 +101,7 @@
                 .withListener(listener)
                 .withPopup(popup);
 
+        this.widgetArgs = widgetArgs;
```

Comment:
This initialization can now happen once, the `final` assignment can be replaced with `this.widgetArgs` and all functions that have that param can use the global value.